src/Gateway/eShop.Gateway/Dockerfile

```diff
@@ -1,6 +1,7 @@
 #See https://aka.ms/customizecontainer to learn how to customize your debug container and how Visual Studio uses this Dockerfile to build your images for faster debugging.
 
 FROM mcr.microsoft.com/dotnet/aspnet:8.0 AS base
+RUN apt-get update && apt-get install -y curl
```

Comment:
Não tenho certeza que isso seja valido. A imagem ja é criada com suas versões corretas e definidas. Atualizar aqui imagino que só incrementa o tempo de geração e não traga nenhum benficio real.

Reply:
O comando `apt-get update` atualiza as informações sobre os repositorios dos pacotes, ele não atualiza nenhum pacote.
A imagem padrão do asp.net não vem com o wget ou com o curl, por isso `apt-get install -y curl` é utilizado, o healthcheck no Docker usa esse comando.

[Docker Healthchecks in ASP.NET](https://learn.microsoft.com/en-us/aspnet/core/host-and-deploy/health-checks?view=aspnetcore-8.0#docker-healthcheck)

Reply:
Ahhh entendi, vc faz isso pra colocar o HelthCheck no proprio Docker... manjei. Concordo 0%, mas não vejo qquer problema em se manter.
Pra que entenda, não concordo por dois motivos.
1 - Em producao, geralmente existe um orquestrador ... ele faz isso automaticamente. Se alguem quiser usar isso (geralmente infra), ela é responsavel pelos yamls.
2 - O health check do .net possui UI especifica pra esses endpoints, que é muito mais vantajoso do que um simples curl -> /health
https://learn.microsoft.com/en-us/dotnet/architecture/microservices/implement-resilient-applications/monitor-app-health

E não vejo qualquer problema em manter, porque apesar de não trazer muitos beneficios, traz o do aprendizado e aqui é o que importa.